pages/profile/read/views/profile.jsx

```diff
@@ -102,7 +95,7 @@ class Profile extends React.Component {
                 isOwnProfile && (
                   <form method='POST' action={`/e/${estId}/projects/create`}>
                     <p className="control-panel">
-                      <Button className='govuk-button add-margin'>
+                      <Button className='govuk-button button-secondary add-margin'>
```

Comment:
not going to block this, as the code was already there, but if you use a `<Button />` you don't need to add the govuk-button class as this is added by default

Reply:
👍 noted